packages/extester/src/browser.ts

```diff
@@ -105,7 +105,7 @@ export class VSBrowser {
 
 		const driverBinary = process.platform === 'win32' ? 'chromedriver.exe' : 'chromedriver';
 		let chromeDriverBinaryPath = path.join(this.storagePath, driverBinary);
-		if (this.codeVersion >= '1.86.0') {
+		if (compareVersions(this.codeVersion, '1.86.0') >= 0) {
```

Comment:
what dou you think about use `satisfies` from a `compare-versions` library? imho it could bring even more readability to do code, wdyt?

`satisfies(this.codeVersion, '>=1.86.0');`

Reply:
Didn't know about that, but it does seems a lot nicer semantically.

Reply:
if I can ask you to use `satisfies` instead, it would be great. Anyways it does not change nothing on that you have spotted a weak spot which is great. Thank you for your time!